src/assets/properties-panel.css

```diff
@@ -1,7 +1,7 @@
 /**
  * Theming
  */
-.bio-properties-panel {
+:root {
```

Comment:
Let's consider to have a shared base class instead, [i.e. `djs-parent`](https://github.com/bpmn-io/diagram-js/blob/develop/assets/diagram-js.css#L4) or something. The alternative is that we prefix our variables (which we don't usually do at bpmn.io).

Without any of these measures we risk to clash with other variables that the user defined. These names sound _very_ common to me.

Reply:
I doubt that we clash since the names are the color spefication (eg `color-grey-225-10-15`). At worst, we would override with the same color. But for safety, I added `djs-parent` (needs to have both that and `bio-properties-panel` since the latter is out of the parent)